### DISCUSSION TITLE
Hiding lines and nodes

### DISCUSSION THREAD

**Discussion by Fury1SOG** [KillBait/AutoDrive_Course_Editor#63] [Ideas]
Hello,

Especially when editing a parking lot for a large garage, there can be a large number of nodes and lines within a small amount of space. It can be very confusing when trying to pick a certain node.
Would it be possible to temporarily hide selected nodes and their corresponding lines?

**Comment by KillBait** [comment]
Strange you should post this, this is one of the things i have been working on lately.

The only downside is that the system i have now is slowing down the speed of the
map renderer, so i'm trying to improve the speed of the map renderer to gain some
( if not all ) of the lost performance back, unfortunately, this is taking a lot longer than
i planned due to all the testing involved to make sure what i do has a positive effect.

If they were hidden completely, i would have to add a keybind to temporarily show them
again so you can see them to un-hide them. I'm not keen on this way of doing it. 

Currently, the system i have now is when i select them as hidden, they are drawn very
transparent ( to be **almost** invisible ) and cannot be selected. Would this work as an
alternative?

**Reply by Fury1SOG** [reply]
Thank you very much for your quick reply!
I would think that nearly transparent and locked nodes/lines would work.

When they are locked, they would not be selectable. Just curious how would they be returned back to editable condition?

**Comment by KillBait** [comment]
The current system is un-hide them to make them editable again.
